createdisk.sh

```diff
@@ -264,7 +252,35 @@ ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo rpm-ostree install /home/
 # Remove the packages from VM
 ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- rm -fr /home/core/hyperv
 
-# Shutdown the instance
+# Shutdown and Start the VM after installing the hyperV daemon packages.
+# This is required to get the latest ostree layer which have those installed packages.
+sudo virsh shutdown ${VM_PREFIX}-master-0
+# Wait till instance started successfully
+until sudo virsh domstate ${VM_PREFIX}-master-0 | grep shut; do
```

Comment:
you need LC_ALL=C or similar, on my machine `domstate` reports `fermé` for a shut off VM.

Reply:
@cfergeau `export LC_ALL=C` and `export LANG=C` area already as part of initial of the script so we don't need for every command.